### ISSUE TITLE
warning html lang attribute 

### ISSUE THREAD

**Issue by littlekelly164** [validator/validator#766]
i get this warning on all my html documents:
Warning: Consider adding a lang attribute to the html start tag to declare the language of this document.

From line 1, column 16; to line 2, column 6

```html
TYPE html>↩<html>↩<head
```

eventho my document start like this:

```html
<!DOCTYPE html>
<html>
```
etc..

my books told me to do it the way i did (lesson 1, made in notepad)

thanks already for looking into this

**Comment by sideshowbarker** [comment]
Your books need to be updated.

Best practice is to always add a `lang` attribute to the `html` element; for example:

```html
<!DOCTYPE html>
<html lang=en>
```

…that indicates the content of the document is in English. Among other things, that helps screen readers choose the right voice and language when reading the content of a document aloud to a user.

**Comment by sideshowbarker** [comment]
Closing per https://github.com/validator/validator/issues/766#issuecomment-466369662
